src/hooks/screen/useChatScreenHook.ts

```diff
@@ -58,6 +63,16 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       setTimeout(() => {
         setLoadingChat(false);
       }, 350);
+
+      if (type === 'ANONYMOUS') {
+        const userInfo = await UserSchema.getSelfAnonUserInfo(
+          localDb,
+          anonProfileId,
+          selectedChannel?.id
+        );
+
+        setSelfAnonUserInfo(userInfo);
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is no error handling for the async function `UserSchema.getSelfAnonUserInfo`. If this function throws an error, it will be unhandled and could potentially crash your application. Consider adding a try-catch block around this function call.
```diff
       if (type === 'ANONYMOUS') {
+        try {
           const userInfo = await UserSchema.getSelfAnonUserInfo(
             localDb,
             anonProfileId,
             selectedChannel?.id
           );
+        } catch (error) {
+          console.error('Error getting self anonymous user info:', error);
+        }
         setSelfAnonUserInfo(userInfo);
       }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->